src/calculator.py

```diff
@@ -15,6 +15,15 @@
 logging.basicConfig(level=logging.INFO)
 
 
+# Handles SIGTERM (15)
+class KillHandler():
```

Comment:
Could you consolidate the three copies of this into a single, separate file and just import it into `calculator.py` `main.py` and `submitter.py`?